### PR TITLE
Replace ColumnFamilies to key prefix for index db

### PR DESCRIPTION
**Pull request by longfin**
- [x] Implement prefix-based chain ID
- [x] Add migration method and command
- [ ] Add support to https://github.com/planetarium/ninechronicles.snapshot

### CODE REVIEW

**Comment by riemannulus** [comment]
Is it support https://github.com/planetarium/ninechronicles.snapshot now?

**Comment by limebell** [comment]
> Is it support https://github.com/planetarium/ninechronicles.snapshot now?

Should be released first in order to be supported in snapshot.

File: CHANGES.md [hunk 1]
```diff
@@ -37,9 +37,13 @@ To be released.
 
 ### CLI tools
 
+  - Added `planet store migrate-index` for index database migration.
```
Comment (riemannulus) [inline CHANGES.md:40]: IMO not only CLI tools logs but also written about `ColumnFamilies`.

File: CHANGES.md [hunk 2]
```diff
@@ -37,9 +41,14 @@ To be released.
...
+    (from column families based to key-prefix)  [[#1838]]
+
 [#1828]: https://github.com/planetarium/libplanet/issues/1828
 [#1831]: https://github.com/planetarium/libplanet/pull/1831
 [#1832]: https://github.com/planetarium/libplanet/pull/1832
+[#1838]: https://github.com/planetarium/libplanet/pull/1838
+[column families]: https://github.com/facebook/rocksdb/wiki/Column-Families
```
Comment (riemannulus) [inline CHANGES.md:51]: This ref is available? I mean, I can't find on the upper comment

Comment (limebell) [reply CHANGES.md:51]: It is referenced in https://github.com/planetarium/libplanet/pull/1838/files#diff-d975bf659606195d2165918f93e1cf680ef68ea3c9cab994f033705fea8238b2R23

**Review by riemannulus — APPROVED** [review]

**Pull request merged**